perf/istio-install/create_cluster.sh

```diff
@@ -154,39 +146,49 @@ function gc() {
 NETWORK_SUBNET="--create-subnetwork name=${CLUSTER_NAME}-subnet"
 # shellcheck disable=SC2236
 set +u
-if [[ -n "${USE_SUBNET}" ]];then
+if [[ -n "${USE_SUBNET:-}" ]];then
   NETWORK_SUBNET="--network ${USE_SUBNET}"
 fi
 set -u
 
 ADDONS="HorizontalPodAutoscaling"
 # shellcheck disable=SC2236
 set +u
-if [[ -n "${ISTIO_ADDON}" ]];then
+if [[ -n "${ISTIO_ADDON:-}" ]];then
   ADDONS+=",Istio"
 fi
 set -u
 
+# Export CLUSTER_NAME so it will be set for the create_sa.sh script, which will
+# create set up service accounts
+export CLUSTER_NAME
+mkdir -p "${WD}/tmp/${CLUSTER_NAME}"
+"${WD}/create_sa.sh" "${GCP_SA}" "${GCP_CTL_SA}"
+
 # shellcheck disable=SC2086
 # shellcheck disable=SC2046
-gc beta container \
-  --project "${PROJECT_ID}" \
-  clusters create "${CLUSTER_NAME}" \
-  --no-enable-basic-auth --cluster-version "${GKE_VERSION}" \
-  --issue-client-certificate \
-  --machine-type "${MACHINE_TYPE}" --image-type ${IMAGE} --disk-type "pd-standard" --disk-size "${DISK_SIZE}" \
-  --scopes "${SCOPES}" \
-  --num-nodes "${MIN_NODES}" --enable-autoscaling --min-nodes "${MIN_NODES}" --max-nodes "${MAX_NODES}" --max-pods-per-node "${MAXPODS_PER_NODE}" \
-  --enable-stackdriver-kubernetes \
-  --enable-ip-alias \
-  --metadata disable-legacy-endpoints=true \
-  ${NETWORK_SUBNET} \
-  --default-max-pods-per-node "${MAXPODS_PER_NODE}" \
-  --addons "${ADDONS}" \
-  --enable-network-policy \
-  --workload-metadata-from-node=EXPOSED \
-  --enable-autoupgrade --enable-autorepair \
-  --labels csm=1,test-date=$(date +%Y-%m-%d),version=${ISTIO_VERSION},operator=user_${USER}
+if [[ "$(gcloud beta container --project "${PROJECT_ID}" clusters list --filter=name="${CLUSTER_NAME}" --format='csv[no-heading](name)')" ]]; then
```

Comment:
should we use `gc` or gcloud?

Reply:
The 'gc' function adds some additional flags that aren't compatible with all commands, which includes this one.